Add JUnit5 and initial test

jkosh44 opened the pull request:
Added configurations to build.gradle for JUnit5. Added trivial tests
for Pair to ensure JUnit is working properly

Addresses Issue #15

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Mojang/DataFixerUpper?pullRequest=22) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Mojang/DataFixerUpper?pullRequest=22) before we can accept your contribution.<br/><br/>:x: jkosh44 [sign now](https://cla.opensource.microsoft.com/Mojang/DataFixerUpper?pullRequest=22)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Mojang/DataFixerUpper?pullRequest=22) it.</sub>

jkosh44 commented:
Thought I'd get the ball rolling on adding tests since I didn't see any test classes or configurations. I added some configurations for JUnit5 to the build.gradle file. I also added two pretty trivial tests for the `Pair` class just to check and see if it was working properly.

RainWarrior commented:
We went with JUnit4 for now since that's what we're using for other projects, will hopefully look into this more later
